crates/pop-cli/src/commands/build/contract.rs

```diff
@@ -1,18 +1,15 @@
 // SPDX-License-Identifier: GPL-3.0
 
 use crate::cli;
-use clap::Args;
 use pop_contracts::{build_smart_contract, Verbosity};
 use std::path::PathBuf;
 
-#[derive(Args)]
+/// Command to build a smart contract with configurable path and release mode.
 pub struct BuildContractCommand {
```

Comment:
It is not a command anymore. Something along the lines of the following might be better:
```
/// Build contract options.
pub struct BuildContract {
```

The fields docs also need to be changed. The `release` field can be as simple as the profile docs of the `BuildParachainCmd`

Reply:
Thanks for the feedback, it doesn't make sense to keep the word Command in the name, or the command description as it was before. Refactored in https://github.com/r0gue-io/pop-cli/pull/394/commits/f1963e7bd59df8820b984bc5ae0aaaa95adaaee4